src/components/button/examples/button-disabled-vs-hidden.tsx

```diff
@@ -0,0 +1,160 @@
+import { Component, h, State } from '@stencil/core';
+import { ListSeparator, MenuItem } from 'src/interface';
+
+/**
+ * Hiding or Disabling Irrelevant Buttons in User Interfaces
+ *
+ */
+@Component({
+    tag: 'limel-example-button-disabled-vs-hidden',
+    shadow: true,
+    styleUrl: 'button-disabled-vs-hidden.scss',
+})
+export class ButtonDisabledVsHideExample {
+    @State()
+    private value: boolean = false;
+
+    @State()
+    private clicked: boolean = false;
+
+    @State()
+    public required: boolean;
+
+    private items: Array<ListSeparator | MenuItem> = [
+        { text: 'Edit', icon: 'save_as' },
+    ];
+
+    public render() {
+        return [
+            <h3>Disabled vs. Hidden</h3>,
+            <p>
+                Effective use of 'Disabled' and 'Hidden' buttons is crucial in
+                user interface design, and the choice between the two should
+                always be context-dependent. These guidelines provide insights
+                into when to apply each approach to optimize user experiences.
```

Comment:
```suggestion
                When should an interactive element –like a button– be "Disabled" and
                when should we avoid displaying it? This is a crucial question in
                user interface design, and the choice between the two should
                always be context-dependent. These guidelines provide insights
                into when to apply each approach to optimize user experiences.
```

Reply:
I'm not sure this rewriting will make any difference :) It sounds the same to me. Or what was the intention of your version?😀

Reply:
the first sentence 😊 
this is not a principle that applies on to "buttons". This is about anything that is interactive, and receives an input from the users. Right? Most of the times, it's about buttons. But we don't have "hidden" buttons really. Hidden is not a property of a button component. 

This guide is about: **_Why you sometimes should NOT simply disable something, and instead choose not to show it._** 

This is also why using "hidden" is a little bit like a miscommunication of the purpose. It's like setting `display: none;` on an element; while the elements shouldn't even have been there in the UI.

Reply:
> this is not a principle that applies on to "buttons".

The idea was to apply this guidance to the button case specifically, that’s why I used only different types of buttons and not other UI elements.

Of course, `disable/hide` could be applied to anything and be more generic.  The reason, why I stick to buttons is because I got the concrete question from our consultants about when to disable and when to hide buttons.  This is quite often confusions from their side. I think we should address it here and be specific.

Reply:
I understand that. 😊  Did you read this comment? https://github.com/Lundalogik/lime-elements/pull/2598#pullrequestreview-1722529492

Reply:
fixed